message_import.go

```diff
@@ -20,7 +20,7 @@ const (
 	maxImportCount = 10
 
 	// MaxImportSize is the maximum total request size permitted for a single import request.
-	MaxImportSize = 30 * 1024 * 1024
+	MaxImportSize = 50 * 1024 * 1024
```

Comment:
So, following our discussion: should we put 50 or 55 here?
Apart from that, good for me.